src/Controller/CRUDController.php

```diff
@@ -1292,6 +1328,33 @@ protected function getBaseTemplate()
      * @throws \Exception
      */
     protected function handleModelManagerException(\Exception $e)
+    {
+        if ($e instanceof ModelManagerThrowable) {
+            $this->handleModelManagerThrowable($e);
+
+            return;
+        }
+
+        @trigger_error(sprintf(
+            'The method %s is deprecated since sonata-project/admin-bundle 3.107 and will be removed in 5.0.',
```

Comment:
IMO, we must keep the responsibility for these changes to the pre-release PRs.
```suggestion
            'The method %s is deprecated since sonata-project/admin-bundle 3.x and will be removed in 5.0.',
```

Reply:
> IMO, we must keep the responsibility for these changes to the pre-release PRs.

What's the reason behind ? In that particular case, I wanted to gain some time

Reply:
I understand your motivation.
I think the reason behind this rule is to keep the changes agnostic to a specific version, as we don't know which release will include them until the PR is merged. By instance, we could decide to release 3.107 before and if this rule is honored, the PR should be still compatible with any future version.

Reply:
Sure, but I think we already merged PR with number of version when the number of version was sure, it's not like there was any other PR for new feature on 3.x

Reply:
I'm gonna merge like this, since I got two approval and don't see a reason to change 3.107 to 3.x to then rechange to 3.107 again if it's ok for you

Reply:
Yes, it isn't a blocker IMO.